### DISCUSSION TITLE
UI Suggestion: Adopt modern Plex client Hub Header style

### DISCUSSION THREAD

**Discussion by millercentral** [pannal/plex-for-kodi#105] [Ideas]
Today PM4K uses an allcaps roughly 20pt type for Hub headers. In current "modern" Plex client layouts, they have changed this to a larger, Title Case type:

![Screenshot 2024-05-09 141126](https://github.com/pannal/plex-for-kodi/assets/7720102/9afb769d-4574-4c52-813b-1c5930be4743)

It seems like these headers are a bold font, approximately 42pt size (or roughly twice the size of regular labels). The headers appear to have about 6px of padding above selected items, or 32px of padding above unselected items.

Personally I prefer the modern Title Case version over the current all caps. Should PM4K consider updating to match the other clients?

**Comment by pannal** [comment]
I think this looks good, but I'd need to see a full screen mockup of this to judge it. Maybe I'll whip something up when I find the spare time. The issue is, we're using caps for all headings and IMHO we should change all of them if we do this. WDYT?

**Comment by millercentral** [comment]
Yeah, page headings don't work quite the same way in the native clients, so there isn't a direct comparison, but maybe it would look better if everything was Title Caps. That said, I don't think Hub Heading style is required to match page headings... You are right that it probably needs a full screen mockup. Library names are already not all caps, so maybe there is precedence there?

**Comment by pannal** [comment]
Maybe, yeah. Do remind me in a week or so, after I got 0.7.8 out, and I'll mock this up.

**Comment by pannal** [comment]
Hmm. Same font size (font12), bold instead of uppercase:
![image](https://github.com/pannal/plex-for-kodi/assets/1359593/3036961f-b5a3-4e97-b69c-61067456a483)

Font13:
![image](https://github.com/pannal/plex-for-kodi/assets/1359593/c7eb2974-53d8-4819-9c87-6456ce4758c4)

Not a fan of the second one, the first one isn't too bad. It doesn't scream "header" to me, though.

**Reply by millercentral** [reply]
Yeah, non-caps at the same size doesn't really seem like a heading. Maybe try `font40_title`,  `font45_title` or even `font52_title`?
